modules/troubleshooting-disabling-autoreboot-mco-console.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operator-issues.adoc
+
+[id="troubleshooting-disabling-autoreboot-mco-console_{context}"]
+= Disabling Machine Config Operator from automatically rebooting by using the console
+
+To avoid unwanted disruptions from changes made by the Machine Config Operator (MCO), you can use the {product-title} web console to modify the machine config pool (MCP) to prevent automatic rebooting after the Operator makes changes to the machine config.
```

Comment:
This should be changed to be the same as the cli doc

```suggestion
To avoid unwanted disruptions from changes made by the Machine Config Operator (MCO), you can modify the machine config pool (MCP) to prevent the MCO from making any changes to nodes in that pool. This prevents any reboots that would normally be part of the MCO update process. 
```